lib/datastore/request.js

```diff
@@ -465,17 +460,13 @@ DatastoreRequest.prototype.delete = function(keys, callback) {
  *     stream instance is returned.
  *
  * @example
- * //-
- * // Where you see `transaction`, assume this is the context that's relevant to
- * // your use, whether that be a Dataset or a Transaction object.
- * //-
```

Comment:
Can you run me through why some of the text of these are changed and others are removed?